<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainEdit</class>
 <widget class="QMainWindow" name="MainEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1327</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="exitBnt">
        <property name="text">
         <string>Normal Screne</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fullBtn">
        <property name="text">
         <string>Full Screne</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGraphicsView" name="view"/>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>221</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>221</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>window's information</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>index</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>51</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>index:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="indexEdit">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>30</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>point</string>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>30</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>x:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>60</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>y:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="horizonPosEdit">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>30</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="verticalPosEdit">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>60</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="addWindowBtn">
         <property name="text">
          <string>add </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="delWindowBtn">
         <property name="text">
          <string>delete</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="saveBtn">
         <property name="text">
          <string>save</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>size</string>
         </property>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>72</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>width:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>72</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>height:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="widthEdit">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>30</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="heightEdit">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>60</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
